docs/src/main/sphinx/connector/filesystem-azure.md

```diff
@@ -0,0 +1,31 @@
+# Filesystem Azure Storage
+
+
+
+:::{list-table} tbd
+:widths: 25, 75
+:header-rows: 1
+
+* - Property
+  - Description
+* - `fs.native-azure.enabled`
```

Comment:
I understand why the property name is what it is today, but trying to put myself in user's shoes. 
If we require users to turn on Azure/S3/GCP filesystem support before they can use given service, the configuration property doesn't need to contain "native-". It would raise less eyebrows if the configuration was

```
fs.azure.enabled=true
<other azure filesystem configs>
````

Reply:
But I dont think thats actually the case .. you can still use all the old stuff for various object storage

Reply:
In the future when we removed the old stuff and just have the new native setup going we can potentially just get rid of the property and have it always on. 

I think now its would be misleading to use 

fs.azure.enabled=false

because azure would still work .. just via the older libraries